c/tskit/tables.c

```diff
@@ -919,19 +918,20 @@ tsk_individual_table_load(tsk_individual_table_t *self, kastore_t *store)
         metadata_schema_length;
 
     read_table_col_t read_cols[] = {
-        { "individuals/flags", (void **) &flags, &num_rows, 0, KAS_UINT32, 0 },
+        { "individuals/flags", (void **) &flags, &num_rows, 0, TSK_FLAG_STORAGE_TYPE,
```

Comment:
Nit - a single "flag" is a bit, I think we should call it ``TSK_FLAGS_STORAGE_TYPE`` to be consistent

Reply:
Yep, good one. Fixed.